django/src/pages/templates/extends/base.html

```diff
@@ -1,56 +1,173 @@
+			</li>
+			<li>
+				<button onclick=toggleSubMenu(this) class="dropdown-btn">
+					<svg xmlns="http://www.w3.org/2000/svg" height="40px" viewBox="0 -960 960 960" width="40px"
+						xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 -960 960 960" width="24px"
+						fill="#000000">
+						<path
+							d="M182-200q-51 0-79-35.5T82-322l42-300q9-60 53.5-99T282-760h396q60 0 104.5 39t53.5 99l42 300q7 51-21 86.5T778-200q-21 0-39-7.5T706-230l-90-90H344l-90 90q-15 15-33 22.5t-39 7.5Zm16-86 114-114h336l114 114q2 2 16 6 11 0 17.5-6.5T800-304l-44-308q-4-29-26-48.5T678-680H282q-30 0-52 19.5T204-612l-44 308q-2 11 4.5 17.5T182-280q2 0 16-6Zm482-154q17 0 28.5-11.5T720-480q0-17-11.5-28.5T680-520q-17 0-28.5 11.5T640-480q0 17 11.5 28.5T680-440Zm-80-120q17 0 28.5-11.5T640-600q0-17-11.5-28.5T600-640q-17 0-28.5 11.5T560-600q0 17 11.5 28.5T600-560ZM310-440h60v-70h70v-60h-70v-70h-60v70h-70v60h70v70Zm170-40Z" />
+					</svg>
+					<span>GAMES</span>
+					<svg xmlns="http://www.w3.org/2000/svg" height="40px" viewBox="0 -960 960 960" width="40px"
+						fill="#000000">
+						<path d="M480-344 240-584l47.33-47.33L480-438.67l192.67-192.66L720-584 480-344Z" />
+					</svg>
+				</button>
+				<ul class="sub-menu">
+					<div>
+						<li>
+							<a href="/pong">
+								<svg xmlns="http://www.w3.org/2000/svg" height="40px" viewBox="0 -960 960 960"
+									width="40px" fill="#000000">
+									<path
+										d="M127.67-159.33 80-206.67l170.73-170.72q32.27-32.28 45.44-80.78 13.16-48.5 13.16-139.83 0-57.32 25.34-112.66Q360-766 410-816q89-89 195.72-101 106.73-12 175.61 57.67 69.34 69.33 57.34 176.66-12 107.34-100.67 196-50 50-105.34 74.67T520-387.33q-91.67 0-140.33 12.33-48.67 12.33-81.34 45L127.67-159.33Zm285-332.67q48.33 48 131 35.67 82.66-12.34 148.33-78 65.92-65.73 78.79-147.36 12.88-81.64-35.46-129.31-48.66-48.67-130.16-36.33Q523.67-835 457.33-769q-65.66 65.67-79.16 147.22-13.5 81.56 34.5 129.78Zm318 452q-62 0-105.67-43.67-43.67-43.66-43.67-105.66T625-295q43.67-43.67 105.67-43.67T836.33-295Q880-251.33 880-189.33T836.33-83.67Q792.67-40 730.67-40Zm0-66.67q35 0 58.83-23.83t23.83-58.83q0-35-23.83-58.84Q765.67-272 730.67-272t-58.84 23.83Q648-224.33 648-189.33t23.83 58.83q23.84 23.83 58.84 23.83Zm0-82.66Z" />
+								</svg>
+								<span>PONG</span>
+							</a>
+						</li>
+						<li>
+							<a href="/tictactoe">
+								<svg xmlns="http://www.w3.org/2000/svg" height="40px" viewBox="0 -960 960 960"
+									width="40px" fill="#000000">
+									<path
+										d="M186.67-120q-27 0-46.84-19.83Q120-159.67 120-186.67v-586.66q0-27 19.83-46.84Q159.67-840 186.67-840h586.66q27 0 46.84 19.83Q840-800.33 840-773.33v586.66q0 27-19.83 46.84Q800.33-120 773.33-120H186.67Zm0-66.67h151v-151h-151v151Zm217.66 0h151.34v-151H404.33v151Zm218 0h151v-151h-151v151ZM186.67-404.33h151v-151.34h-151v151.34Zm217.66 0h151.34v-151.34H404.33v151.34Zm218 0h151v-151.34h-151v151.34Zm-435.66-218h151v-151h-151v151Zm217.66 0h151.34v-151H404.33v151Zm218 0h151v-151h-151v151Z" />
```

Comment:
whats is that ?

Reply:
SVG path for defining icons
Changed in commit : [d24d348](https://github.com/TheSecretOrganization/ft_transcendence/pull/201/commits/d24d34875c8598f849f6a6c2f6e5f2b83ab06290)